srv/components/provisioner/scripts/support

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+import re
+import argparse
+import logging
+import os
+import sys
+import datetime
+import yaml
+import subprocess
+from pathlib import Path
+from cortx.utils.security import cipher
+from provisioner.utils import (
+    run_subprocess_cmd,
+    load_yaml
+)
+from provisioner.errors import SubprocessCmdError
+from cortx_setup.config import SUPPORT_USER_NAME
+
+#SUPPORT_CONFIG = Path('/var/data/support.yaml')
+SUPPORT_CONFIG = Path(f'/opt/seagate/users/{SUPPORT_USER_NAME}/support.yaml')
+#logging.basicConfig(
```

Comment:
Lets enable logging for tracking purposes

Reply:
Sure. Done.